test/e2e/nsx_security_policy_test.go

```diff
@@ -186,36 +180,33 @@ func testSecurityPolicyMatchExpression(t *testing.T) {
 	ruleName := "expression-policy-1-rule"
 
 	var err error
-	setupTest(t, ns)
-	defer teardownTest(t, ns, defaultTimeout)
+	testData.createVCNamespace(ns)
```

Comment:
Question: is it possible we deploy a CI testbed not using WCP (VC is not used)? If yes, we may not create namespace on VC, but directly on K8s?

Reply:
Yes, the previous k8s testbed is what you said, running for a while.